src/go/nerdctl-stub/main_shared.go

```diff
@@ -64,7 +64,7 @@ func mountArgProcessor(arg string, mounter func(string) (string, error)) (string
 
 // builderCacheProcessor implements the details for handling the argument for
 // `nerdctl builder build --cache-from=...` and
-// `nerdctl builder builder --cache-to=...`
+// `nerdctl builder build --cache-to=...`
```

Comment:
notable. check-spelling didn't like this, but I don't think nerdctl did either...